public/template_tags.php

```diff
@@ -542,7 +550,7 @@ function espresso_event_date_range( $date_format = '', $time_format = '', $singl
 			$time_format = ! empty( $time_format ) ? $time_format : get_option( 'time_format' );
 			$time_format = apply_filters( 'FHEE__espresso_event_date_range__time_format', $time_format );
 			$html = sprintf(
-				__( '%1$s - %2$s', 'event_espresso' ),
+				esc_html__( '%1$s - %2$s', 'event_espresso' ),
```

Comment:
maybe add the translation context so that translators understand what this is for, either:

- via code comment like: `/* translators: 1: earliest event date, 2: latest event date */`
- or via `esc_html_x()`